datadog_checks_dev/datadog_checks/dev/tooling/config.py

```diff
@@ -53,6 +53,8 @@
                                 ('api_key', os.getenv('DD_API_KEY')),
                                 ('app_key', os.getenv('DD_APP_KEY')),
                                 ('site', os.getenv('DD_SITE')),
+                                ('dd_url', os.getenv('DD_DD_URL')),
+                                ('dd_log_url', os.getenv('DD_LOGS_CONFIG_DD_URL')),
```

Comment:
Mind if we get rid of `dd_`?

Reply:
Sure, keeping it for `dd_url` as it matches the name in the config.